services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    hall = CinemaHall.objects.get(id=cinema_hall_id)
+
+    session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=hall
+    )
+    return session
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession | QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)

```

Comment:
Consider handling the potential `DoesNotExist` exception when retrieving a `MovieSession` object. This will prevent the application from crashing if an invalid ID is provided.